src/api/cli/integration/status.golden

```diff
@@ -0,0 +1,6 @@
+{
+    "running": true,
+    "num_of_blocks": 181,
```

Comment:
number of blocks is 181 now?

Reply:
Yes, I think so. The block of seq 180 is exist in the db.